admin/javascript/src/sections/campaign-admin/list.js

```diff
@@ -79,12 +81,44 @@ class CampaignListContainer extends SilverStripeComponent {
               {accordionGroups}
             </Accordion>
           </div>
+          <div className="cms-south-actions">
+            {this.renderButtonToolbar()}
+          </div>
         </div>
         { previewUrl && <CampaignPreview previewUrl={previewUrl} /> }
       </div>
     );
   }
 
+  renderButtonToolbar() {
+    const items = this.getItems(this.props.campaignId);
```

Comment:
It looks like the `this.getItems` method doesn't use the param it's being passed here.